cli/src/generate/rules.rs

```diff
@@ -70,6 +68,98 @@ pub enum Rule {
     Seq(Vec<Rule>),
 }
 
+impl fmt::Display for Rule {
```

Comment:
I don't really think we should we should have this code for converting a rule back into JavaScript.

IMO, we should remove the warning.